SECURITY.md

```diff
@@ -0,0 +1,5 @@
+# Reporting Security Vulnerabilities
+
+The Chan Zuckerberg Initiative takes security issues seriously. We appreciate your efforts to responsibly disclose your findings, and will make every effort to acknowledge your contributions.
```

Comment:
Nit: in order to use a contraction to connect two clauses, both clauses need to have a subject. `, and will make` -> `, and we will make`.

Reply:
TIL, thanks 🤓